### DISCUSSION TITLE
Drawer from top/bottom

### DISCUSSION THREAD

**Discussion by bjornhansen** [saadeghi/daisyui#3161] [Ideas / feature requests]
For mobile designs, having a drawer open from the bottom or top is useful. Is this on the roadmap, or currently supported but not documented? Seems like a reasonably easy implementation?

Thanks y'all!

**Comment by saadeghi** [comment]
Yes it would be useful.  
It's not supported yet but for now you can use this example:  
https://play.tailwindcss.com/NBg7Q0zzWp

**Reply by bjornhansen** [reply]
Awesome, thank you! Is this is something you'd accept a pull request for?

**Reply by saadeghi** [reply]
Yes but more important than the code, is the decisions about structure, class names, flexibility. If it would be better to have a different name or for it to be a variant of drawer. If it should be done with dialog or popover API.

These are the decisions I'm not sure about yet.

**Reply by bjornhansen** [reply]
Understood, and thank you for your work on DaisyUI. I am very much enjoying it. ❤️

**Comment by erikdemarco** [comment]
@saadeghi  still no update about this feature? Seems like uncomplete to have only left and right. without any top/bottom

**Comment by saadeghi** [comment]
@erikdemarco It will be in daisyUI 5. I'm working hard on version 5 and there are a lot of exciting things included!  
Until then, you can use the example I linked.

**Reply by bjornhansen** [reply]
Exciting! Looking forward to v5, thanks for your work on it!

I appreciated your example by the way.

**Comment by JoeEarly** [comment]
Hi @saadeghi 

did this make it in to v5? 

Thanks :)

**Comment by pdanpdan** [comment]
It can be done (https://play.tailwindcss.com/KUyzQcM5NC?file=css) but from my POV it's not worth it - too much code and the main case of the drawer (open by default on large screens and overlayed on small screens) does not make sense for top/bottom.

Is there a specific case where a bottom dialog does not do exactly what you need? (https://play.tailwindcss.com/TyFPA3jgSY)

**Reply by JoeEarly** [reply]
I got a version of the bottom dialog working and does what I need

Thank you for the response
